pkg/secrethub/http/http.go

```diff
@@ -1,4 +1,4 @@
-package secrethub
+package http
```

Comment:
Consider moving this package to `internals/http` as it is far from ready to be published and we don't want to spend too much time polishing it now.

Reply:
Can also be `pkg/secrethub/internals/http` if we want it to be a subpackage of `pkg/secrethub`. 

Then it's also actually enforced that this can only be consumed by the secrethub package.